### PR TITLE
New applet name fix

### PR DESCRIPTION
**Pull request by wildkiomi**

### CODE REVIEW

**Review by nmialik — APPROVED** [review]

File: src/features/AppletSettings/ShareAppletSetting/ShareAppletSetting.tsx [hunk 1]
```diff
@@ -14,13 +14,19 @@ import { StyledHeadline } from '../AppletSettings.styles';
...
   const [sharePopupVisible, setSharePopupVisible] = useState(false);
   const [keywords, setKeywords] = useState<string[]>([]);
   const [libraryUrl, setLibraryUrl] = useState('');
+  const [applet, setApplet] = useState<FolderApplet | null>(null);
+
+  useEffect(() => {
+    if (id) {
+      setApplet(folders.useApplet(id));
+    }
+  });
```
Comment (IhorNesterenko99) [inline src/features/AppletSettings/ShareAppletSetting/ShareAppletSetting.tsx:29]: maybe id needs to be added to array of dependencies

File: src/features/AppletSettings/TransferOwnershipSetting/TransferOwnershipSetting.tsx [hunk 2]
```diff
@@ -15,18 +15,24 @@ import { StyledTransferOwnershipForm } from './TransferOwnershipSetting.styles';
...
     if (emailTransfered) {
       setTransferOwnershipPopupVisible(true);
     }
   }, [emailTransfered]);
 
+  useEffect(() => {
+    if (id) {
+      setApplet(folders.useApplet(id));
+    }
+  });
```
Comment (IhorNesterenko99) [inline src/features/AppletSettings/TransferOwnershipSetting/TransferOwnershipSetting.tsx:34]: maybe id needs to be added to array of dependencies

File: src/features/Applet/ShareApplet/ShareApplet.tsx [hunk 3]
```diff
@@ -52,42 +52,44 @@ export const ShareApplet = ({
...
-          }),
-        );
+            params: { keywords: JSON.stringify(keywords) },
+          });
+          const libraryUrlResult = await getAppletLibraryUrlApi({
+            appletId: applet.id || '',
+          });
+
+          setLibraryUrl(libraryUrlResult?.data as string);
+          setIsLoading(false);
```
Comment (kdanilenka32) [inline src/features/Applet/ShareApplet/ShareApplet.tsx:77]: по идее можно через finally setLoading(false)

File: src/features/AppletSettings/ShareAppletSetting/ShareAppletSetting.tsx (hunk 1 above)
Comment (kdanilenka32) [reply src/features/AppletSettings/ShareAppletSetting/ShareAppletSetting.tsx:29]: and where the dep array?)

**Review by IhorNesterenko99 — APPROVED** [review]

**Pull request merged**